phpstan.neon

```diff
@@ -1,16 +1,44 @@
+        - '#Strict comparison using === between string and null will always evaluate to false\.#'
+        - '#Cannot call method .+ on .+\|null\.#'
+        - '#Parameter \#1 \$v of method .+Spy.+ expects .+ given\.#'
+        - '#Parameter \#1 \$data of method .+fromArray\(\) expects array\<string, mixed\>, array\|string given\.#'
+        - '#Parameter .+ of function str_replace expects array\|string, string\|null given\.#'
+        - '#Parameter .+ of function strtolower expects string, string\|null given\.#'
+        - '#Parameter .+ of function count expects array\|Countable, array\|null given\.#'
+        - '#Parameter .+ of function substr expects string, string\|null given\.#'
+        - '#Parameter .+ of function ucfirst expects string, string\|null given\.#'
+        - '#Parameter .+ of function strpos expects string, string\|null given\.#'
+        - '#Parameter .+ of function base64_decode expects string, string\|null given\.#'
+        - '#Parameter .+ of function md5 expects string, string\|null given\.#'
+        - '#Parameter .+ of method .+ expects Orm\\Zed\\Payone\\Persistence\\SpyPaymentPayone, Orm\\Zed\\Payone\\Persistence\\SpyPaymentPayone\|null given\.#'
+        - '#Parameter .+ of method .+ expects .+Interface, .+AbstractRequestContainer given\.#'
+        - '#Parameter .+ of method .+ expects .+AbstractRequestContainer, .+Interface given\.#'
+        - '#Parameter .+ of method .+ expects .+Interface, .+Interface given\.#'
+        - '#Parameter .+ of method .+ expects .+AbstractPaymentMethodContainer, .+ContainerInterface given\.#'
+        - '#Parameter .+ of method .+ expects Generated\\Shared\\Transfer\\QuoteTransfer, Generated\\Shared\\Transfer\\QuoteTransfer\|null given\.#'
+        - '#Property SprykerEco\\Zed\\Payone\\Business\\Api\\Call\\CreditCardCheck::\$storeCardData is never read, only written\.#'
+        - '#Parameter .+ of method .+ expects Generated\\Shared\\Transfer\\SaveOrderTransfer, Generated\\Shared\\Transfer\\SaveOrderTransfer\|null given\.#'
+        - '#Parameter .+ of method .+ expects Generated\\Shared\\Transfer\\PayonePaymentTransfer, Generated\\Shared\\Transfer\\PayonePaymentTransfer\|null given\.#'
+        - '#Parameter .+ of method .+ expects Generated\\Shared\\Transfer\\PaymentDetailTransfer, Generated\\Shared\\Transfer\\PaymentDetailTransfer\|null given\.#'
         - '#Call to method .+ on an unknown class SprykerEco\\Zed\\Payone\\Business\\Payment\\RequestSender\\CaptureContainerInterface\.#'
         - '#Parameter \$requestContainer of method .+mapProductData\(\) has invalid type .+ContainerInterface\.#'
         - '#Parameter \$container of method .+preparePartialCaptureOrderItems\(\) has invalid type .+ContainerInterface\.#'
         - '#Call to method .+ on an unknown class Generated\\.+\\Ide\\AutoCompletion\.#'
+        - '#Argument of an invalid type .+ supplied for foreach, only iterables are supported\.#'
+        - '#Method .+ should return .+ but returns .+\.#'
```

Comment:
This one is way too generic, this is not possible to keep as is here.

Reply:
Okey, I'll try to improve.